app/models/installment_purchase.rb

```diff
@@ -1,13 +1,14 @@
 class InstallmentPurchase < ApplicationRecord
   belongs_to :group
-  validates :description, presence: true
-  validates :total_amount, presence: true
-  validates :installment_amount, presence: true
-  validates :installments_quantity, presence: true
   validates :current_installment, presence: true
+  validates :description, presence: true
   validates :due_day, presence: true
-  validates :active, presence: true
   validates :group_id, presence: true
+  validates :installment_amount, presence: true
+  validates :installments_quantity, presence: true
+  validates :status, presence: true
+  validates :total_amount, presence: true
```

Comment:
_Minor suggestion:_ Maybe it'd be nice to validate that it should be a number, greater than zero, etc